client/dashboard/task-list/tasks/payments/payfast.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Link } from '@woocommerce/components';
+import { ADMIN_URL } from '@woocommerce/wc-admin-settings';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'lib/tracks';
+
+export default class PayFast extends Component {
+  constructor( props ) {
+    super( props );
+  }
+
+  continue = () => {
+    recordEvent( 'tasklist_payment_connect_method', {
+      payment_method: 'payfast'
+    } )
+  }
+
+  render() {
+    const link = <Link
+      href={ `${ ADMIN_URL }/admin.php?page=wc-settings&tab=checkout&section=wc_gateway_payfast` }
+      target="_blank"
+      type="external"
+    />;
+    const helpLink = <Link
+      href="https://docs.woocommerce.com/document/payfast-payment-gateway/"
+      target="_blank"
+      type="external"
+    />;
+    const configureText = interpolateComponents( {
+      mixedString: __(
+        'PayFast can be configured under your {{link}}store settings{{/link}}. Figure out {{helpLink}}what you need{{/helpLink}}.',
```

Comment:
For the other payment gateways, we've been adding the minimum required fields to get the gateway working.  What do you think about adding the merchant ID, key, and passphrase here so we can get things running without leaving this page?

/cc @pmcpinto in case you have any input on the copy.

Reply:
The ticket said to use the same approach as Klarna which just has this redirection to the extension settings, but I'm happy to give this a try.

Reply:
Ok, this is done in the last two commits. :successkid: